### PR TITLE
Audio Framework README

### PR DESCRIPTION
**Pull request by btheosam**
Add guide on how to add audio support for an embedded board on Tizen RT.

### CODE REVIEW

**Comment by btheosam** [comment]
Prveiw available in 
https://github.com/btheosam/TizenRT/blob/master/docs/HowToAddAudioSupport.md

File: docs/HowToAddAudioSupport.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+# SECTION I: Tizen RT's Audio Framework
```
Comment (sunghan-chang) [inline docs/HowToAddAudioSupport.md]: # How to add audio support

## Audio Framework
blah blah

## How to add audio for a board
blah blah

File: docs/HowToAddAudioSupport.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# SECTION II: How To Add Audio for a Board
+From SECTION I, changes for a new board are limited to #2 & #4. ie,   
+a. Adding control and data path  
+b. Adding BSP/firmware for audio codec  
+  
+Right now, Tizen RT supports alc5658 audio codec ONLY; through I2C (control path) & I2S (data path).
+
+# SECTION III: Control Path (I2C)
+
+1. os/include/tinyara/i2c.h has data structures and APIs of I2C
```
Comment (sunghan-chang) [inline docs/HowToAddAudioSupport.md]: 1. Is this mentioned in audo doc? I think you had better assume there is a i2c driver and say how to connect i2c to audio.
2. Does all of audio hw use i2c as a control path?

**Comment by sunghan-chang** [comment]
commit title:
```
docs: Add how to add audio support

blah blah
```

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToAddAudioSupport.md (hunk 2 above)
Comment (btheosam) [reply docs/HowToAddAudioSupport.md]: 1. This is not a i2c driver. 
2. This document neve says that all audio hw use i2c as control path

File: docs/HowToAddAudioSupport.md (hunk 1 above)
Comment (btheosam) [reply docs/HowToAddAudioSupport.md]: No. SECTION is needed here. Coz is has to be referred later.

**Comment by btheosam** [comment]
All comments are replied.

**Comment by sunghan-chang** [comment]
@btheosam Could you update this? with peripheral?

**Comment by btheosam** [comment]
Will start with I2C, & I2S. Once those tow are done,should come back to this.

**Pull request closed without merging**